Bottom: 99f08a54ec427ae3e38b3e34abf5dfab5ab5dd1e
Top:    dad9aee1332c84f2544c63b111d77697f9cb5146
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 04:24:33 +0100

Remove function SearchScreenGridSlop() and use SearchObjectByLocation() directly


---

diff --git a/src/crosshair.c b/src/crosshair.c
index 02b76e3..1165e06 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -1292,9 +1292,9 @@ FitCrosshairIntoGrid (Coord X, Coord Y)
       if (TEST_FLAG (RUBBERBANDFLAG, PCB))
         {
 
-          ans = SearchScreenGridSlop (Crosshair.X, Crosshair.Y,
-                                      LINE_TYPE,
-                                      &ptr1, &ptr2, &ptr3);
+          ans = SearchObjectByLocation (LINE_TYPE,
+                                        &ptr1, &ptr2, &ptr3,
+                                        Crosshair.X, Crosshair.Y, PCB->Grid / 2);
           if (ans != NO_TYPE) {
             double angle;
             int octant;
